q2_moshpit/plugin_setup.py

```diff
@@ -379,6 +379,25 @@
                 'analyses.'
 )
 
+plugin.methods.register_function(
+    function=q2_moshpit.eggnog.fetch_eggnog_db,
+    inputs={},
+    parameters={},
+    outputs=[("eggnog_db", ReferenceDB[Eggnog])],
+    output_descriptions={
+        "eggnog_db": "Artifact containing the eggNOG annotation "
+                     "database (eggnog.db, eggnog.taxa.db, "
+                     "eggnog.taxa.db.traverse.pkl)"
+    },
+    name="Fetch one of the databases necessary to run eggNOG.",
+    description="Downloads eggnog reference database  "
+                "using the `download_eggnog_data.py` script from eggNOG. "
+                "Here, this script downloads 3 files, specifically: "
+                "eggnog.db (39Gb), "
+                "eggnog.taxa.db (265Mb), and"
+                "eggnog.taxa.db.traverse.pkl (6.3Mb)."
```

Comment:
Again, let's not mention the sizes here (or anywhere else) as they may change in the future. Maybe you can say instead that the entire database requires at least 40 GB of storage space? (or however much you think it does)

Reply:
I need 80 since it needs to copy the files once everything is downloaded :)